android/tangram/src/com/mapzen/tangram/MapController.java

```diff
@@ -229,6 +230,26 @@ public void loadSceneFile(String path) {
     }
 
     /**
+     * Load a new scene file
+     * @param path Location of the YAML scene file within the application assets
+     * @param sceneUpdates List of path:value pairs to be applied when loading this scene
+     */
+    public void loadSceneFile(String path, ArrayList<SceneUpdate> sceneUpdates) {
```

Comment:
These methods should take a `List<SceneUpdate>`, the `List` interface should be preferred to concrete types when we don't need the specific type.